src/Compilers/CSharp/Portable/FlowAnalysis/CSharpDataFlowAnalysis.cs

```diff
@@ -102,28 +103,45 @@ public override ImmutableArray<ISymbol> DataFlowsIn
         }
 
         /// <summary>
-        /// The set of local variables which are definitely assigned a value when a region
-        /// is entered.
-        /// 
-        /// The set of variables that are definitely assigned a value when a region is exited is
-        /// the union of <see cref="DefinitelyAssignedOnEntry"/> and <see cref="AlwaysAssigned"/>.
+        /// The set of local variables which are definitely assigned a value when a region is
```

Comment:
What is a region here?

Reply:
the dataflowanalysis apis are passed a span of the source code to ask questions about.  commonly, this is the span around an expression, a single statement, or a pair of start/end statements.